### DISCUSSION TITLE
Tools

### DISCUSSION THREAD

**Discussion by giscus** [bids-standard/bids-website#407] [General]
# Tools

https://bids-website.readthedocs.io/en/latest/tools/index.html

**Comment by bendhouseart** [comment]
First Post on Tools Page! (revised)

**Comment by Remi-Gau** [comment]
The landing page of the tool section should have brief description of the different sub sections when can find in it.

Very confusing for people on mobile who otherwise have to open the navbar to discover the content.

**Comment by Remi-Gau** [comment]
The "other" tools could probably be split further: annotation tool, "query" tools (pybids, bids-matlab...)

**Reply by Remi-Gau** [reply]
Done

**Comment by Remi-Gau** [comment]
Other tools could benefit of a badge showing the last commit (for maintainers alos knowing the test status of each tool)

**Reply by Remi-Gau** [reply]
Done

**Comment by Remi-Gau** [comment]
split PET and MRI converters on different pages

**Comment by Remi-Gau** [comment]
MRI converters should mention what dataype they can do: some converter specialize in some datatype (ASL for example)
